pyuvdata/uvdata/uvdata.py

```diff
@@ -8143,8 +8208,26 @@ def _select_preprocess(
                     "(optionally with polarization) or a list of baseline numbers."
                 )
             if not all(
-                [isinstance(item[0], (int, np.integer,),) for item in bls]
-                + [isinstance(item[1], (int, np.integer,),) for item in bls]
+                [
+                    isinstance(
+                        item[0],
+                        (
+                            int,
+                            np.integer,
+                        ),
+                    )
+                    for item in bls
+                ]
+                + [
+                    isinstance(
+                        item[1],
+                        (
+                            int,
+                            np.integer,
+                        ),
+                    )
+                    for item in bls
+                ]
```

Comment:
I think this is too hard to read and needs to be refactored in some way to make it better.

Reply:
Fixed in b5ea2cbd.